README.md

```diff
@@ -1005,6 +1030,23 @@ The `not` and `case: ignore` directives can both be applied on the same rule. Th
   - [ X-Forwarded-For, { value: a, not: prefix, case: ignore } ]
 ```
 
+#### meta headers
+In addition to the directives that check the actual content of individual
+fields, Proxy Verifier supports meta-fields denoted by an `@` prefix that serve
+as directives for the tool to perform custom operations. The following
+demonstrates the usage of the `@hasfields` meta-field to verify that the
+`header` node has at least one field:
```

Comment:
This seems to imply there are multiple supported `@` meta headers. But I think just `@hasfields` is supported, right? I might add as a second sentence:

> Currently, the only meta header that is supported is `@hasfields` is supported which can verify that there are or are not header fields with a `proxy-request` or `proxy-response`.